src/SamplesApp/SamplesApp.UITests/TestFramework/ImageAssert.cs

```diff
@@ -15,159 +15,110 @@ namespace SamplesApp.UITests.TestFramework
-								.AppendLine($"Pixels at ({x}, {y}) are not the same")
-								.AppendLine($"expectedColor: {ToArgbCode(expectedColor)} {expectedColor}")
-								.AppendLine($"actualColor  : {ToArgbCode(actualColor)} {actualColor}")
-							));
-						}
-					}
-			}
-
-			string WithContext(string message = null, Action<StringBuilder> builder = null)
-			{
-				return new StringBuilder()
-					.AppendLine($"ImageAssert.AreEqual @ line {line}")
-					.AppendLine("expected: " + expected?.Name + (expectedRect == FirstQuadrant ? null : $"in {expectedRect}"))
-					.AppendLine("actual  : " + actual?.Name + (actualRect == FirstQuadrant ? null : $"in {actualRect}"))
-					.AppendLine("====================")
-					.ApplyIf(message != null, x => x.AppendLine(message))
-					.ApplyIf(builder != null, builder)
-					.ToString();
-			}
-		}
-		#endregion
-
-		#region AreAlmostEqual
 		/// <summary>
 		/// Asserts that two screenshots are equal to each other inside the area of the nominated rect to within the given pixel error
 		/// (ie, the A, R, G, or B values cumulatively differ by more than the permitted error for any pixel).
 		/// </summary>
 		public static void AreAlmostEqual(FileInfo expected, FileInfo actual, IAppRect rect, int permittedPixelError, [CallerLineNumber] int line = 0)
-			=> AreAlmostEqualImpl(expected, rect.ToRectangle(), actual, rect.ToRectangle(), permittedPixelError, line);
+			=> AreEqualImpl(expected, rect.ToRectangle(), actual, rect.ToRectangle(), 1, PixelTolerance.Cummulative(permittedPixelError), line);
 
 		/// <summary>
 		/// Asserts that two screenshots are equal to each other inside the area of the nominated rect to within the given pixel error
 		/// (ie, the A, R, G, or B values cumulatively differ by more than the permitted error for any pixel).
 		/// </summary>
 		public static void AreAlmostEqual(FileInfo expected, FileInfo actual, Rectangle? rect = null, int permittedPixelError = 5, [CallerLineNumber] int line = 0)
-			=> AreAlmostEqualImpl(expected, rect ?? FirstQuadrant, actual, rect ?? FirstQuadrant, permittedPixelError, line);
+			=> AreEqualImpl(expected, rect ?? FirstQuadrant, actual, rect ?? FirstQuadrant, 1, PixelTolerance.Cummulative(permittedPixelError), line);
 
 		/// <summary>
 		/// Asserts that two screenshots are equal to each other inside the area of the nominated rect to within the given pixel error
 		/// (ie, the A, R, G, or B values cumulatively differ by more than the permitted error for any pixel).
 		/// </summary>
 		public static void AreAlmostEqual(FileInfo expected, IAppRect expectedRect, FileInfo actual, IAppRect actualRect, int permittedPixelError, [CallerLineNumber] int line = 0)
-			=> AreAlmostEqualImpl(expected, expectedRect.ToRectangle(), actual, actualRect.ToRectangle(), permittedPixelError, line);
+			=> AreEqualImpl(expected, expectedRect.ToRectangle(), actual, actualRect.ToRectangle(), 1, PixelTolerance.Cummulative(permittedPixelError), line);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to add validation of parameter 'actualRect' before using it.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5363624)